### PR TITLE
[README] Correct some broken grammar & spelling.

### PR DESCRIPTION
**Pull request by shlomif**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,18 +15,19 @@ uutils coreutils
...
 
 Why?
 ----
 
-Many GNU, Linux and other utils are pretty awesome, and obviously
+Many GNU, Linux and other utilities are useful, and obviously
 [some](http://gnuwin32.sourceforge.net) [effort](http://unxutils.sourceforge.net)
 has been spent in the past to port them to Windows. However, those projects
-are either old, abandoned, hosted on CVS, written in platform-specific C, etc.
+are either old, abandoned, hosted on CVS, written in platform-specific C, or
```
Comment (Arcterus) [inline README.md:27]: I’d probably change `old, abandoned` to `old/abandoned`.  Also I feel like we should clarify that `hosted on CVS` is a negative as it makes it more difficult for new contributors.

File: README.md [hunk 2]
```diff
@@ -15,18 +15,19 @@ uutils coreutils
 -----------------------------------------------
 
 uutils is an attempt at writing universal (as in cross-platform) CLI
-utils in [Rust](http://www.rust-lang.org). This repo is to aggregate the GNU
-coreutils rewrites.
+utilities in [Rust](http://www.rust-lang.org). This repository is intended for
+aggregating the GNU coreutils rewrites.
```
Comment (Arcterus) [inline README.md:19]: This sounds kinda weird to me.  I’d prefer `intended to aggregate GNU coreutils rewrites` or something like that.

**Comment by shlomif** [comment]
@Arcterus : thanks for reviewing the changes! I'll try to revise my commit tomorrow since it's getting late here.

**Comment by shlomif** [comment]
@Arcterus : I revised the changeset based on your commentary and rebased. Sorry it took me so long - I woke up late and it slipped off my mind.

BTW, your family name reminds me of this jamendo band: https://www.jamendo.com/artist/440873/lyonn . I like their "We'll light the sky" and "Shining Eyes" songs (YKMV).

**Comment by Arcterus** [comment]
No worries, a day isn’t very long, and I was preoccupied anyway.

I’ll give them a listen. :)

**Comment by shlomif** [comment]
Thanks, @Arcterus !

**Pull request merged**